msocasecharge.yaml

```diff
@@ -0,0 +1,176 @@
+        column: "OFFENSE_DESCRIPTION"
+      ol.datetimestart:
+        type: "ol.datetimestart"
+        column: "OFFENSE_DATE"
+        transforms:
+        - !<transforms.DateTimeTransform>
+          pattern: ["yyyy-MM-dd HH:mm:ss", "yyyy-MM-dd HH:mm:ss.S", "yyyy-MM-dd HH:mm:ss.SS", "yyyy-MM-dd HH:mm:ss.SSS"]
+          timezone: "America/New_York"
+      date.completeddatetime:
+        type: "date.completeddatetime"
+        column: "DATE_ADDED"
+        transforms:
+        - !<transforms.DateTimeTransform>
+          pattern: ["yyyy-MM-dd HH:mm:ss", "yyyy-MM-dd HH:mm:ss.S", "yyyy-MM-dd HH:mm:ss.SS", "yyyy-MM-dd HH:mm:ss.SSS"]
+          timezone: "America/New_York"
+      ol.notes:
+        type: "ol.notes"
+        column: "COMMENTS"
+    name: "chargeevent"
+
+  disposition:
+    fqn: "ol.disposition"
+    entitySetName: "MSODisposition"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "DISPOSITION"
+      criminaljustice.disposition:
+        type: "criminaljustice.disposition"
+        column: "DISPOSITION"
+      general.datetime:
+        type: "general.datetime"
+        column: "DISPOSITION_DATE"
+        transforms:
+        - !<transforms.DateTimeTransform>
+          pattern: ["yyyy-MM-dd HH:mm:ss", "yyyy-MM-dd HH:mm:ss.S", "yyyy-MM-dd HH:mm:ss.SS", "yyyy-MM-dd HH:mm:ss.SSS"]
+          timezone: "America/New_York"
+    name: "disposition"
+
+  bond:
+    fqn: "ol.bond"
+    entitySetName: "MSOBond"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "BOND_AMOUNT"
+      criminaljustice.bailbondamount:
+        type: "criminaljustice.bailbondamount"
+        column: "BOND_AMOUNT"
+    name: "bond"
+
+  case:
+    fqn: "justice.case"
+    entitySetName: "MSOCase"
+    propertyDefinitions:
+      j.CaseNumberText:
+        type: "j.CaseNumberText"
+        column: "CASE_PK"
+      justice.courtcasenumber:
+        type: "justice.courtcasenumber"
+        column: "CASE_PK"
+    name: "case"
+
+associationDefinitions:
+  relatedto:
+    fqn: "ol.relatedto"
+    entitySetName: "MSORelatedto"
+    src: "charge"
+    dst: "jailstay"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "assn_relatedto"
+    name: "relatedto"
+
+  registeredfor:
+    fqn: "ol.registeredfor"
+    entitySetName: "MSORegisteredfor"
+    src: "chargeevent"
+    dst: "charge"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "assn_registeredfor"
+    name: "registeredfor"
+
+  bondset:
+    fqn: "ol.bondset"
+    entitySetName: "MSOBondset"
+    src: "jailstay"
+    dst: "bond"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "assn_bondset"
+    name: "bondset"
+
+  appearsin:
+    fqn: "ol.appearsin"
+    entitySetName: "MSOAppearsin"
+    src: "jailstay"
+    dst: "case"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "assn_appearsin"
+    name: "appearsin"
+
+  clearedby:
```

Comment:
Also need charge event -> cleared by -> disposition and  person -> charged with -> charge event